clients/cobol-lsp-vscode-extension/README.md

```diff
@@ -9,7 +9,7 @@
 
 # COBOL Language Support
 
```

Comment:
Should I remove this doc change from this PR?

Reply:
Do you mean publishing them as a separate PR?

Reply:
Yes, changes for README in a separate PR.

Reply:
Yes, please. This is a semantically separate change.